### ISSUE TITLE
WireGuard-ESP32-Arduino works on an Inkplate!

### ISSUE THREAD

**Issue by rspencer01** [ciniml/WireGuard-ESP32-Arduino#5]
Not an issue, per-se, but rather a note to say that this library works flawlessly (for my needs) on an [Inkplate 10](https://www.crowdsupply.com/e-radionica/inkplate-10).  Thank you!  

Arduino gives the following warning though:
```
WARNING: library WireGuard-ESP32 claims to run on esp32 architecture(s) and may be incompatible with your current board which runs on Inkplate architecture(s)
```
However, I have found it to work fine.

Is there any reason in the code that this should not be the case?  Are there any ESP32 specific things that might not work in the Inkplate harness?  I'd take a look through the code, but don't really have time at the moment to learn its inner workings.

If not, can I suggest adding Inkplate to the list of supported architectures?

**Comment by ciniml** [comment]
@rspencer01 I've released v0.1.4 of this library, which supports `Inkplate` as a platform. Please try this after Arduino library is published from the arduino library repository.

**Comment by rspencer01** [comment]
Happy to report that Arduino no longer warns of this issue.  Thanks for the quick response!
